Systematics/test/workspaceStd_thq.py

```diff
@@ -0,0 +1,736 @@
+#!/usr/bin/env cmsRun
```

Comment:
Is this cfg used for debugging purposes? If yes it might be worth to move it to Validation/test

Reply:
also it looks mostly as a copy and paste from wrokspaceStd.py, so maybe you can have the features implemented there with a debug tHq switch